src/FeedsFactory.php

```diff
@@ -19,6 +19,12 @@ public function make($feed_url) {
     $this->simplepie = new SimplePie();
     $this->configure();
     $this->simplepie->set_feed_url($feed_url);
+	if ( ! $this->config['strip_html_tags.disabled'] && ! empty( $this->config['strip_html_tags.tags'] ) && is_array($this->config['strip_html_tags.tags'])) {
```

Comment:
Can you extract this to a better named function so anyone can understand in one sentence what it does?